DEVELOPING.md

```diff
@@ -4,25 +4,14 @@
 
 When creating a pull request, you must verify that your changes are safe by running them on a large codebase such as [https://github.com/kubernetes/kubernetes](https://github.com/kubernetes/kubernetes). This is not intended as a replacement for proper automated testing. Running without error on a large codebase provides an additional level of confidence, since a large codebase is likely to contain edge cases that you may not have considered. Indeed, such edge cases have caused failures in the past ([#74](https://github.com/google/go-flow-levee/pull/74), [#143](https://github.com/google/go-flow-levee/pull/143)).
 
-Here's an example script showing how you can test your changes against [Kubernetes](https://github.com/kubernetes/kubernetes):
+In addition we recommend running master and feature branch versions of levee against k8s and diffing the results to see if your changes have led to some new findings. To make this easier we have created a script that will run both master and your feature branch version of levee against k8s and print the diff.
 
-```bash
-#!/bin/bash
-
-set -eux
-
-# set these to appropriate values for your setup
-# GO_FLOW_LEVEE_PATH=path to the root of the go-flow-levee repository on your machine
-# K8S_PATH="$GOPATH"/src/k8s.io/kubernetes
+Note: The script makes the following assumptions
+1. k8s repo is at $(GOPATH)/src/k8s.io/kuberenetes
```

Comment:
A few nits:
* kube*rn*etes
* $(GOPATH) ... should probably be in backticks
* I think a full sentence would read better, e.g. "The kubernetes repository on your machine is at ..."

Reply:
Done.